splash/qtutils.py

```diff
@@ -254,171 +256,380 @@ def swap_byte_order_i32(buf):
     return arr.tostring()
 
 
+def render_qwebpage(web_page, logger=None, width=None, height=None,
```

Comment:
What do you think about moving all render-related functions and classes into a dedicated module? Something like qtutils_render.py.